minio/minio
# add min length validation to access key and secret key

## Description

While changing the credentials through MinIO browser,
Update button will be disabled if keys are lesser than
minimum length.

Fixes #7713

## Motivation and Context


Update button on the Change password modal should remain
disabled if access key or secret key entered is smaller than
the minimum required length.

## Regression


No

## How Has This Been Tested?



1. Open Change Password dialog on MinIO Browser
2. Enter less than 3 chars in new access key, Update button should remain disabled
3. Enter less than 8 chars in new secret key, Update button should remain disabled

## Types of changes


## Checklist:
